### ISSUE TITLE
No React Display

### ISSUE THREAD

**Issue by richwolf817** [benawad/fullstack-graphql-airbnb-clone#30]
Hey,

Think I've done everything correctly, after starting the server and web just not seeing anything in browser. No errors on Chrome console. 

Thanks,
Rich
<img width="1680" alt="Screen Shot 2019-08-31 at 2 15 30 PM" src="https://user-images.githubusercontent.com/32384198/64070391-1371d300-cbfa-11e9-9611-ae5843dbb879.png">

**Comment by benawad** [comment]
https://github.com/benawad/fullstack-graphql-airbnb-clone/issues/26

**Comment by richwolf817** [comment]
Cool thanks! So I'm not getting the confirmation emails. Is there something I need to do to get that working locally?

**Comment by benawad** [comment]
It'll be in the server logs https://youtu.be/jsrtxnFpGaw?t=424

**Comment by oristides** [comment]
I really don know what to do here #26 to show something.

**Comment by benawad** [comment]
@oristides go to one of the routes

**Comment by oristides** [comment]
Yes, I saw the route and the file. But how can I have the webapp on air
locally?
Sorry I'm noob. I did the tutorial but I really don't get it.

**Comment by benawad** [comment]
https://github.com/benawad/fullstack-graphql-airbnb-clone#installation

**Comment by oristides** [comment]
Ok, I followed all install steps.

But when I yarn start at packages/server and then i yarn start at ackages/web.

Then it opens a white page 

http://localhost:3000/

You told something about a file in Routes https://github.com/benawad/fullstack-graphql-airbnb-clone/blob/master/packages/web/src/routes/index.tsx But I don' t know how to continue from this point.

**Comment by benawad** [comment]
look at the path for the pages you can go to `path="/register"`

**Comment by sangdth** [comment]
I think it's good to mention that we don't have the index page in the README. I'm confused in the first place too.

**Comment by stevegiorgi** [comment]
@benawad thanks for all your work on this.  I followed your instructions and had to modify them slightly for Windows 10.  Did not run into any hitches at all, but I would not have thought of checking the routes in a million years without stumbling onto this thread.  I was also met by a blank white screen once yarn build was processed.  I think for new users just dipping into React such as myself, it would have save a lot of time to simply mention that the index page is blank and to follow the routes to navigate the application.  Thanks again Ben.
